config/settings.yml.example

```diff
@@ -30,7 +30,9 @@
 # Uncomment and modify if you want to change the default pid file '/var/run/foreman-proxy/foreman-proxy.pid'
 #:daemon_pid: /var/run/foreman-proxy/foreman-proxy.pid
 
-# HTTP ports configuration
+# host and ports configuration
+# host to bind ports to (default: 0.0.0.0 for production, localhost for development)
```

Comment:
Ok so I think we agree that default value is always `0.0.0.0` for production so I will remove the comment and just leave there the default commented value. Development is not that important and people can just override.